### ISSUE TITLE
"mac" not detected as os

### ISSUE THREAD

**Issue by jdx** [houseabsolute/ubi#79]
```
$ ubi -p wren-lang/wren-cli
[ubi][ERROR] could not find a release for this OS (aarch64-apple-darwin) from wren-cli-linux-0.4.0.zip, wren-cli-mac-0.4.0.zip, wren-cli-windows-0.4.0.zip
```

**Comment by autarch** [comment]
I have a fix for this in the `more-tests` branch I just pushed. Thanks for the report!

**Comment by autarch** [comment]
Thanks for the bug report! This should be fixed in v0.2.4.
